src/recipe.cpp

```diff
@@ -444,28 +440,67 @@ std::string recipe::required_skills_string( const Character *c, bool print_skill
     } );
 }
 
-std::string recipe::required_skills_string( const Character *c ) const
-{
-    return required_skills_string( c, false );
-}
-
-std::string recipe::required_skills_string() const
+// Format a std::pair<skill_id, int> for the basecamp bulletin board.
+// skill colored white with difficulty in parenthesis.
+template<typename _FIter>
```

Comment:
Similarly here.

Reply:
Note that I based this off existing code:

https://github.com/CleverRaven/Cataclysm-DDA/blob/master/src/output.h#L690